### PR TITLE
Imported cafe: fix add project to collections bug

### PR DESCRIPTION
**Pull request by sarahzinger**
## Links
* https://imported-cafe.glitch.me
* https://glitch.manuscript.com/f/cases/3328144/

## How To Test:
* To recreate the bug: go to a collection, click add project to collection and search for a url for a project that doesn't exist (https://glitch.com/~nothinghereatall works for me), notice in the console, an error fires off
* notice on my remix that does not in fact happen! yay!

### CODE REVIEW

File: server/api.js [hunk 1]
```diff
@@ -3,7 +3,6 @@
 const axios = require('axios');
 const { Cache } = require('memory-cache');
 const dayjs = require('dayjs');
-const punycode = require('punycode');
```
Comment (sarahzinger) [inline server/api.js]: i didn't do this but someone else must have on the live project or something and it just didn't get captured on github. I think it's chill, seeing as we don't use that variable in this file.

**Comment by modernserf** [comment]
hmmm i thought this looked familiar 💩  https://github.com/FogCreek/Glitch-Community/pull/514/files

**Comment by modernserf** [comment]
but i'm glad that you actually provided steps to reproduce

**Comment by sarahzinger** [comment]
@modernserf LOL I'll close mine and assign you the ticket my bad!

**Pull request closed without merging**